HelpSource/Tutorials/A-Practical-Guide/PG_03_What_Is_Pbind.schelp

```diff
@@ -222,7 +222,11 @@ p = Pbind(
 p.stop;
 ::
 
-Prior to 3.5, the convention to include a rest in the middle of an event pattern is to set the code::\freq:: key to a link::Classes/Symbol::. Commonly this is code::\rest::, but a backslash by itself is enough to suppress the note on the server. (This usage is still supported in 3.5, but not recommended.)
+note::
+In 3.9, it is no longer supported to use the Rest class in patterns. All rests need to have a value, e.g. code::Rest(0)::. Rest objects now support math operators. That is, you can now write code::Pseq([1, 2, Rest(0)], inf) * 2:: emphasis::and:: code::[1, 2, Rest(0)] * 2::. (Prior to 3.9, the former usage was supported, but only in Pbind, and the latter usage was not supported at all.)
+::
+
+The older convention for rests is to set the code::\freq:: key to a link::Classes/Symbol::. Commonly this is code::\rest::, but a backslash by itself is enough to suppress the note on the server. (This usage is still supported in 3.5, but not recommended.)
```

Comment:
Should this be updated to say 3.9 instead of 3.5?

Reply:
Hm, yeah. 3.5 is a holdover from the initial introduction of Rest in that version. Will do later.

Reply:
alright! otherwise I think it looks good. thanks for catching this :)